fe/src/main/java/org/apache/doris/http/meta/ColocateMetaService.java

```diff
@@ -54,8 +58,11 @@
 
     private static ColocateTableIndex colocateIndex = Catalog.getCurrentColocateIndex();
 
-    private static long checkAndGetGroupId(BaseRequest request) throws DdlException {
-        long groupId = Long.valueOf(request.getSingleParameter(GROUP_ID).trim());
+    private static GroupId checkAndGetGroupId(BaseRequest request) throws DdlException {
+        long grpId = Long.valueOf(request.getSingleParameter(GROUP_ID).trim());
+        long dbId = Long.valueOf(request.getSingleParameter(DB_ID).trim());
+        GroupId groupId = new GroupId(dbId, grpId);
+
```

Comment:
`palo/fe/src/main/java/org/apache/doris/http/Http API Reference` this file seems need to update?

Reply:
Yes，I will do it later~

Reply:
can you move this file to our documents?

Reply:
It will be added later.